py2many/cli.py

```diff
@@ -392,6 +397,9 @@ def main(args=None, env=os.environ):
     parser.add_argument(
         "--extension", type=bool, default=False, help="Build a python extension"
     )
+    parser.add_argument(
+        "--no-prologue", type=bool, default=False, help=""
```

Comment:
IMO it would make sense to broaden the scope of this a bit, and also exclude the cpplint rule exception markup, and other type of superfluous annotations being added to the generated code.